.coveragerc

```diff
@@ -0,0 +1,7 @@
+# .coveragerc explained
+#     - https://coverage.readthedocs.io/en/coverage-4.2/config.html
+#     - https://coverage.readthedocs.io/en/coverage-4.2/source.html#source
+
+[run]
+omit =
+    *_compat.py
```

Comment:
this is here because CI is set up to fail if tests don't meet a coverage threshold, and the try-catched import stuff in `_compat.py` is impossible to 100% cover